src/QueryM.purs

```diff
@@ -81,34 +85,8 @@ import Data.Traversable (traverse, for)
 import Data.Tuple.Nested ((/\))
 import Data.UInt (UInt)
 import Data.UInt as UInt
-import DatumCacheWsp
-  ( DatumCacheMethod
-      ( StartFetchBlocks
-      , CancelFetchBlocks
-      , DatumFilterAddHashes
-      , DatumFilterRemoveHashes
-      , DatumFilterSetHashes
-      )
-  , DatumCacheRequest
-      ( GetDatumByHashRequest
-      , GetDatumsByHashesRequest
-      , StartFetchBlocksRequest
-      , CancelFetchBlocksRequest
-      , DatumFilterAddHashesRequest
-      , DatumFilterRemoveHashesRequest
-      , DatumFilterSetHashesRequest
-      , DatumFilterGetHashesRequest
-      )
-  , DatumCacheResponse
-      ( GetDatumByHashResponse
-      , GetDatumsByHashesResponse
-      , DatumFilterGetHashesResponse
-      )
-  , faultToString
-  , requestMethodName
-  , responseMethod
-  )
-import DatumCacheWsp as DcWsp
+import QueryM.DatumCacheWsp (DatumCacheMethod(StartFetchBlocks, CancelFetchBlocks, DatumFilterAddHashes, DatumFilterRemoveHashes, DatumFilterSetHashes), DatumCacheRequest(GetDatumByHashRequest, GetDatumsByHashesRequest, StartFetchBlocksRequest, CancelFetchBlocksRequest, DatumFilterAddHashesRequest, DatumFilterRemoveHashesRequest, DatumFilterSetHashesRequest, DatumFilterGetHashesRequest), DatumCacheResponse(GetDatumByHashResponse, GetDatumsByHashesResponse, DatumFilterGetHashesResponse))
```

Comment:
This line is extremely long. Although this should be moot soon since we're adding a purs-tidy config with a line length limit